docs/openapi.rst

```diff
@@ -73,8 +73,8 @@ illustrates how to pass `summary` and `description` using that decorator.
 
 .. code-block:: python
 
-    @blp.doc('description': 'Return pets based on ID',
-             'summary': 'Find pets by ID')
+    @blp.doc({'description': 'Return pets based on ID',
+             'summary': 'Find pets by ID'})
```

Comment:
Thanks. There is a typo indeed but this is the wrong fix.

The `doc` decorator expects `kwargs`.

https://github.com/Nobatek/flask-rest-api/blob/master/flask_rest_api/blueprint.py#L224

```py
                @blp.doc(description="Return pets based on ID",
                         summary="Find pets by ID"
                )
```

Reply:
fixed it